containers/podman-desktop
# chore: correct port validation

### What does this PR do?

The following changes request provides better port validation process for Run Images dialog.

### Screenshot / video of UI

<img width="1162" src="https://github.com/containers/podman-desktop/assets/1968177/3d56e85f-e003-4daf-a4de-ef6ebd95fece">
<img width="1162" src="https://github.com/containers/podman-desktop/assets/1968177/40e4fbb9-e0e9-42a0-8fcb-7eb7eb6d0c92">
<img width="1162" src="https://github.com/containers/podman-desktop/assets/1968177/9e215de8-cf16-44b3-8865-ad31b0a07038">
<img width="1162" src="https://github.com/containers/podman-desktop/assets/1968177/a78be5bc-bf31-41d5-a804-28a98b0ac283">
<img width="1162" src="https://github.com/containers/podman-desktop/assets/1968177/daec0ef0-7ba2-4c1c-a84a-a2cd321c4f5a">

### What issues does this PR fix or reference?

#6390

### How to test this PR?

Try to create two different containers with same exposed port to check that new port is proposed in Run Image dialog and validation of input works proper.
